viewer/core/static/js/app/components/containers/dataset-control-container.js

```diff
@@ -29,10 +36,13 @@ class DatasetControlContainer extends React.Component {
     const selectedDatasetId = parseInt(evt.target.parentNode.querySelector('.dataset-selection').value, 10);
     urlApi.updateQueryParameter('ds', selectedDatasetId);
 
-    const selectedSubgroups = Array.from(
-        document.body.querySelectorAll('.dataset-cohorts .switch.active')
-    ).map(ss => ss.parentNode.textContent);
-    urlApi.updateQueryParameter('sg', selectedSubgroups.join(','));
+    processSubgroups();
```

Comment:
Not a big deal, but in theory do we still need this if the toggles have an immediate effect?